react/components/LogoutCustomerSession.tsx

```diff
@@ -49,6 +53,24 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm();
+
+  const id = orderForm.id;
```

Comment:
```suggestion
  const { id } = orderForm
```

Reply:
Updated the code to use object destructuring

Reply:
@lariciamota Updated the code to use object destructuring